expression.dd

```diff
@@ -1691,10 +1681,20 @@ $(GNAME AssertExpression):
     $(P When compiled and run, it will produce the message:
 
 $(CONSOLE
-Error: AssertError Failure test.d(3) an error message
+core.exception.AssertError@test.d(3) an error message
 )
     )
 
+    $(P The expression $(D assert(0)) is a special case; it
+        marks code as unreachable.
+        Either $(D AssertError) is thrown at runtime if it is reachable,
```

Comment:
Instead of "Either", please document how the `-release` flag influences this behavior.